app/src/main/java/org/oppia/app/player/SplitScreenManager.kt

```diff
@@ -0,0 +1,44 @@
+package org.oppia.app.player
+
+import android.app.Activity
+import android.util.DisplayMetrics
+import org.oppia.app.R
+import kotlin.math.pow
+import kotlin.math.sqrt
+
+private const val MINIMUM_DIAGONAL_WIDTH = 7.0
+
+class SplitScreenManager(
+  private val activity: Activity
+) {
```

Comment:
Checkout `OppiaClock` or `OppiaDateTimeFormatter` to convert this file into dagger injection.

Reply:
Done